tssc/step_implementers/tag_source/git.py

```diff
@@ -164,7 +166,7 @@ def _run_step(self, runtime_step_config):
                 password = runtime_step_config.get('password')
             else:
                 raise ValueError('Both username and password must have ' \
-                  'non-empty value in the runtime step configuration')
+                                 'non-empty value in the runtime step configuration')
```

Comment:
this whitespacing looks off, should be 2 spaces or a tab

Reply:
@kfrankli this is the way linter likes it, with it alligned with the start of the start of the string. to prevent so much whitespacei normally put the first line on a new line, then it only needs to be indented once, and then the second line only needs to be indented once as well.

Both of these are valid
````
    raise ValueError('Both username and password must have ' \
                     'non-empty value in the runtime step configuration')
```
or
````
    raise ValueError(
        'Both username and password must have ' \
        'non-empty value in the runtime step configuration')
```